src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -107,8 +106,8 @@ object EasyStageDataset {
       _ <- EasyStageFileItem.createFileSdo(sdoDir, relativePath, "objectSDO" -> parentSDO
       )(FileItemSettings(
         sdoSetDir = Some(s.sdoSetDir),
-        pathInStorage = Some(new File(relativePath)),
-        storageBaseUrl = s.bagStorageLocation,
+        dsLocation = None,
```

Comment:
ds? Don't make me guess. Use abbreviations sparingly. Do not involved developers know everything from the top of their heads. Only a few sources later that spelled out datastream I understood.